driver/src/main/java/org/neo4j/driver/internal/metrics/spi/PoolStatus.java

```diff
@@ -20,5 +20,18 @@
 
 public enum PoolStatus
 {
-    Open, Closed
+    Open( 0 ),
```

Comment:
I'd vote for renaming enum values to `OPEN` and `CLOSED` which follows the recommended java code style